### PR TITLE
Containerize v1-content-annotator and v1-annotations-rw.

### PR DESCRIPTION
**Pull request by BerniVarga**

### CODE REVIEW

**Comment by tosan88** [comment]
:+1:

File: v1-annotations-rw-sidekick@.service [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+BindsTo=v1-annotations-rw@%i.service
+After=v1-annotations-rw@%i.service
+
+[Service]
+ExecStart=/bin/sh -c "\
+  etcdctl set   /ft/services/v1-annotations-rw/healthcheck     true; \
+  etcdctl mkdir /ft/services/v1-annotations-rw/servers; \
+  etcdctl set /ft/healthcheck/v1-annotations-rw-%i/path /__health; \
+  while true; do \
+    export V1_ANNOTATIONS_RW_PORT=$(/usr/bin/docker port v1-annotations-rw-%i 8080 | cut -d':' -f2); \
```
Comment (tamas-molnar) [inline v1-annotations-rw-sidekick@.service]: please use optimized way of setting the port

File: v1-content-annotator-sidekick@.service [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Description=V1 Content Annotator Service Sidekick
+BindsTo=v1-content-annotator@%i.service
+After=v1-content-annotator@%i.service
+[Service]
+ExecStart=/bin/sh -c "\
+  etcdctl set   /ft/services/v1-content-annotator/healthcheck     true; \
+  etcdctl mkdir /ft/services/v1-content-annotator/servers; \
+  etcdctl set /ft/healthcheck/v1-content-annotator-%i /__health; \
+  while true; do \
+    export V1_CONTENT_ANNOTATOR_PORT=$(/usr/bin/docker port v1-content-annotator-%i 8080 | cut -d':' -f2); \
```
Comment (tamas-molnar) [inline v1-content-annotator-sidekick@.service]: please use optimized way of setting the port

File: v1-annotations-rw-sidekick@.service (hunk 1 above)
Comment (BerniVarga) [inline v1-annotations-rw-sidekick@.service]: done

File: v1-content-annotator-sidekick@.service (hunk 2 above)
Comment (BerniVarga) [inline v1-content-annotator-sidekick@.service]: done

**Comment by tamas-molnar** [comment]
+1

**Comment by martingartonft** [comment]
+1

**Comment by kovacshuni** [comment]
+1

**Pull request merged**